src/e2e/specs/landing.spec.ts

```diff
@@ -124,18 +108,23 @@ test.describe("Landing Page Functionality Verification", () => {
       // verify second CTA button
       await expect(scanPage.signUpForAlerts).toBeVisible();
 
-      // verify redirect to sign in("/user/breaches")
+      // verify redirect to sign in('/user/breaches')
       expect(await scanPage.signUpForAlerts.getAttribute("href")).toBe(
         "/user/breaches",
       );
       expect(
         await scanPage.getAlertsAboutBreachesButton.getAttribute("href"),
       ).toBe("/user/breaches");
 
-      // verify "have i been pwned" website redirect
+      // verify 'have i been pwned' website redirect
       expect(await scanPage.haveIBeenPwnedLink.getAttribute("href")).toBe(
         "https://haveibeenpwned.com/",
       );
-    }).toPass();
+    } else {
+      console.log(
+        "[e2e] - Timeout reached without being redirect to the free scan results page",
+      );
+      expect(page.url().includes("/scan")).toBeTruthy();
```

Comment:
Is this actually reason for a successful test?

Reply:
this fails if the monitor app was not able to successfully redirect to a free scan page

Reply:
We already know that it wasn't able to sucessfully redirect, because `freeScanRedirect` is `false`, no? So if this tests the same thing, couldn't we just move this to line 98, and skip the whole `if .. else` block?